### DISCUSSION TITLE
How to modify players

### DISCUSSION THREAD

**Discussion by biggiecheesetherat** [Techokami/SonicWorldsNext#215] [Q&A] [answered]
Hey all! I'm wondering how I can add/remove playable characters in this engine. Is there a way I can do such? Thanks :)

**Comment by Raptorq1** [comment, marked as answer]
Yes there is just check in the Global script in that script you will find a array named CHARACTERS that's where you would add the character to

**Reply by biggiecheesetherat** [reply]
Ok, thank you :)
